build.sbt

```diff
@@ -34,13 +40,15 @@ val HadoopDependencies = Seq(
 )
 
 val TestDependencies = Seq(
-  "org.scalatest" %% "scalatest" % ScalaTestVersion % "test->*",
-  "org.scalamock" %% "scalamock-scalatest-support" % ScalamockVersion % "test->*"
+  scalatest,
+  scalacheck,
+  scalamock,
+  scalactic
```

Comment:
Those are not test dependencies anymore, as far as I can tell. I look like you made `compile` also depend on them

Reply:
Good catch. @adrianmatwiejuk Could you take a look?